compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -199,7 +219,7 @@ use crate::errors::{
 };
 
 #[derive(PartialEq)]
-pub enum MonoItemCollectionMode {
+pub enum MonoItemCollectionStrategy {
```

Comment:
I renamed this to "strategy" since we now also have the `CollectionMode` (used vs mentioned) and I could not come up with a better name for the new enum.